regression/cbmc-path/Makefile

```diff
@@ -0,0 +1,19 @@
+default: tests.log
```

Comment:
Please add this directory to `regression/Makefile` and `regression/CMakeLists.txt` -- although I'm not sure a new directory is strictly required, it might be safe to include that test in the cbmc/ folder?